content/en/moved.html

```diff
@@ -0,0 +1,41 @@
++++
+title = "Voter Registration After a Move"
+translationKey = "about"
+aliases = [
+  "/moved"
+]
```

Comment:
You can remove this. It is only necessary when the filename is different than the path you want the page to be located.